core/db/queries.py

```diff
@@ -26,13 +26,43 @@ def filter_on_regions(itl_regions: set[str]) -> list[UUID]:
     return updated_results
 
 
+def query_extend_with_outcome_filter(base_query, outcome_categories: list[str] | None = None):
```

Comment:
Can you add Query as type hint here please? Can do as per `download_data_base_query`

Reply:
for both the 1st param, and the return type. Ta